.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description
+What does this PR do?
+
+### TODO
+- [ ] Make sure there's a changelog. Brand new features have a differently
+  formatted changelog than other changelogs, so pay attention to that. If this
+  PR is the CE portion of an ENT PR, put the changelog here, _not_ in your ENT
+  PR.
+- [ ] Add a milestone. If this change is going to be backported, each backport
+  PR should receive the milestone for the exact version it's being backported
+  to. For this PR, just pick the biggest version.
+- [ ] If this PR is the CE portion of an ENT change, and that ENT change is
+  getting backported to N-2, use the new style `backport/ent/x.x.x+ent` labels
+  instead of the old style `backport/x.x.x` labels.
+- [ ] If this PR is a CE only change, it can only be backported to N, so use
+  the normal `backport/x.x.x` label (there should be only 1).
+- [ ] If this PR either 1) removes a public function OR 2) changes the signature
```

Comment:
Do you think it would be helpful to link to the Confluence page? https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/2295365741/The+Vault+Development+Cycle#Enterprise-or-OSS

This raises another question: do we want this to apply to community-submitted PRs? The only way I can think of making a distinction would be to instead push the logic into CI, but that doesn't seem great.

Reply:
This is a HC employee only thing. Community members can't check the ENT repo. The Confluence page doesn't seem like it adds much clarity to me. In my mind, this should be a self-explanatory checklist that people can just eyeball and check off. If we're requiring background reading to complete the checklist, it's a bad checklist and people probably won't follow it.

That said, if this item doesn't feel self-explanatory to you, or should be a in a greater level of detail, I'm certainly open to re-wording it.